resources/views/welcome.blade.php

```diff
@@ -78,13 +81,12 @@
             @lang('main.open')</a><br class="mobile-break" />
     </div>
 
-    @if (config('app.debug'))
-        <script>
-            $(document).ready(function(){
-                $('.tooltipped').tooltip();
-            });
-        </script>
-    @endif
+    <script>
+        $(document).ready(function(){
+            $('.tooltipped').tooltip();
+            $(".dropdown-trigger").dropdown();
+        });
+    </script>
```

Comment:
This is broken; it complains about `$` being undefined (it's live on staging, see there)

Can this be done without jQuery? Also, feels odd to have a dropdown for this on desktop.

cc @kdmnk

Reply:
mine